jaas/src/main/java/com/griddynamics/jagger/jaas/storage/model/JobEntity.java

```diff
@@ -29,8 +31,8 @@
     private Long jobStartTimeoutInSeconds;
 
     @JsonIgnore
-    @OneToOne(mappedBy = "job", cascade = CascadeType.REMOVE)
-    private JobExecutionEntity jobExecutionEntity;
+    @OneToMany(mappedBy = "job", fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<JobExecutionEntity> jobExecutionEntities;
```

Comment:
is the design correct here? job execution entities should know about jobs. why should jobs know about job execution entities?